### PR TITLE
add 'user-agent' in HTTP header

### PR DESCRIPTION
**Pull request by zunsthy**
https://github.com/feross/webtorrent/issues/962

### CODE REVIEW

File: lib/client/http-tracker.js [hunk 1]
```diff
@@ -89,8 +89,14 @@ HTTPTracker.prototype._request = function (requestUrl, params, cb) {
   var self = this
   var u = requestUrl + (requestUrl.indexOf('?') === -1 ? '?' : '&') +
     common.querystringStringify(params)
+  var opts = {
+    url: u,
+    header: {
+      'user-agent': self.client._userAgent || ''
```
Comment (feross) [inline lib/client/http-tracker.js]: I don't see where `self.client._userAgent` actually gets set. Did you forget to commit changes to `client.js`?

File: lib/client/http-tracker.js [hunk 2]
```diff
@@ -89,8 +89,14 @@ HTTPTracker.prototype._request = function (requestUrl, params, cb) {
   var self = this
   var u = requestUrl + (requestUrl.indexOf('?') === -1 ? '?' : '&') +
     common.querystringStringify(params)
+  var opts = {
+    url: u,
+    header: {
```
Comment (feross) [inline lib/client/http-tracker.js]: Also, this should be `headers`.

**Comment by feross** [comment]
8.1.0

File: lib/client/http-tracker.js (hunk 2 above)
Comment (zunsthy) [inline lib/client/http-tracker.js]: Sorry, I ignored it. Thank you for correction : )

**Pull request merged**